notifier.js

```diff
@@ -1,54 +1,169 @@
 var config = require('./config'),
-    request = require('request');
+    async = require('async'),
+    request = require('request'),
+    logger = require('winston');
 
 var db = require(config.database.type);
+var acc_modules = {};
 
 /**
- * Send the accounting information to the WStore.
+ * Send the usage specification for the unit passed to the Usage Managament API.
  *
- * @param  {string} apiKey              Identifies the product.
- * @param  {Object} notificationInfo    Information for notify the WStore.
+ * @param  {string}   unit     Accounting unit.
  */
-exports.notify = function (notificationInfo, callback) {
-    var body = {
-        customer: notificationInfo.customer,
-        timestamp: (new Date()).toISOString(),
-        value: notificationInfo.value,
-        correlationNumber: notificationInfo.correlationNumber,
-        recordType: notificationInfo.recordType,
-        unit: notificationInfo.unit
-    };
-
-    db.getToken(function (err, token) {
-        if (err) {
-            return callback('Error obtaining the token');
-        } else {
+var sendSpecification = function (unit, callback) {
+    acc_modules[unit].getSpecification(function (specification) {
+        db.getToken(function (err, token) {
             var options = {
-                url: config.WStore.url + notificationInfo.orderId + '/' + notificationInfo.productId,
+                url: 'http://' + config.usageAPI.host + ':' + config.usageAPI.port + config.usageAPI.path + '/usageSpecification',
                 json: true,
                 method: 'POST',
                 headers: {
                     'Content-Type': 'application/json',
-                    'X-API-KEY': token
+                    authorization: token
```

Comment:
This header is not correct, the authorization header need to specify the type, in this case Bearer:

Authorization: 'Bearer ' + token